backend/src/main/java/gov/cdc/usds/simplereport/api/model/accountrequest/WaitlistRequest.java

```diff
@@ -7,8 +7,12 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import lombok.Getter;
+import lombok.Setter;
 
 @JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
+@Getter
+@Setter
```

Comment:
went ahead and just used the lombok annotations to clean up this file

Reply:
Ohhh this looks much better now -- thank you 😇